streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -609,6 +609,7 @@ private KafkaStreams createKafkaStreams(String topic, final CountDownLatch latch
 
                     @Override
                     public void init(ProcessorContext context) {
+                        super.init(context);
```

Comment:
Since we don't use the context anyway, an alternative fix is to use `Processor` directly rather than `AbstractProcessor`.